### DISCUSSION TITLE
For You page error

### DISCUSSION THREAD

**Discussion by wanjohiryan** [community/community#13824] [Other Feature Feedback, Questions, & Ideas] [answered]
For you page keeps showing 'error reload page'. I keep reloading, but it won't budge.

**Comment by William-McGonagle** [comment]
Same here...

**Comment by jfullstackdev** [comment, marked as answer]
hopefully that's fixed by now, never had that on my Chrome, even just one single moment

and 

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
